Cohort2/team4/client/src/pages/_app.tsx

```diff
@@ -1,12 +1,16 @@
 import '@/styles/globals.css'
 import Layout from '../components/layout'
+import NavBar from '@/components/NavBar'
+import Footer from '@/components/Footer'
 
 import type { AppProps } from 'next/app'
 
 export default function App({ Component, pageProps }: AppProps) {
   return (
 			<Layout>
+				<NavBar />
```

Comment:
The purpose of the Layout component is to hold all of the common features of our pages. So ideally the <NavBar /> and <Footer /> would be a part of the layout, so that its included in every page. Can you move these over to there?